README.md

```diff
@@ -144,12 +144,12 @@ const reader = stream.getReader();
 // Listen for changes
 while (true) {
 	// Read from the stream
-	const {done, notification} = await reader.read();
-
-	// Do something with each notification
-	console.log(notification);
+	const { done, value } = await reader.read();
```

Comment:
The value, in this case, is actually a live query notification.

Reply:
I'm not a JS guy so I realise that notification might actually have a different meaning.